lib/services/ecf/output_payment_aggregator.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Services
```

Comment:
```suggestion
module PaymentCalculator
```

Can we update the namespace to something that better describes this area?

Reply:
done, good call

Reply:
(specs still to do)

Reply:
specs done